src/components/events/EventModal/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Typography } from '@/components/common';
+import CommunityLogo from '@/components/common/CommunityLogo';
```

Comment:
```suggestion
import { CommunityLogo } from '@/components/common';
```
super tiny nit: try to import from the enclosing folder to ensure imports are always named correctly